webcomponents/tool/src/main/frontend/js/sakai-i18n.js

```diff
@@ -24,7 +24,12 @@ function loadProperties(suppliedOptions) {
     return;
   }
 
-  const lang = window.parent.portal && window.parent.portal.locale ? window.parent.portal.locale : "";
+  // https://stackoverflow.com/questions/19565776/check-if-window-parent-is-same-domain
+  let lang = "";
+  try {
+    lang = window.parent.portal && window.parent.portal.locale ? window.parent.portal.locale : "";
+  } catch (e) { }
```

Comment:
log message would be ideal here

Reply:
This is not my code and not a good idea to fix deep inside this PR.